### DISCUSSION TITLE
withoutSaveHistory

### DISCUSSION THREAD

**Discussion by ialiev685** [ianstormtaylor/slate#5476] [General]
Hello. I need help. I want to delete a tag without being able to undo the last action. How do I do that? I do it with the withoutSaveHistory method. Then when I click "undo last change" an error comes up.

**Comment by charanjit-singh** [comment]
Can you share the error?

**Reply by ialiev685** [reply]
![image](https://github.com/ianstormtaylor/slate/assets/75394796/ceb3801c-3b7a-4f2a-b71b-65f439638526)


I do not save in the history of the creation and deletion of the tag. I believe that the program can not restore the data. How to properly describe the process so that you can not restore tags?
